src/main/java/com/epam/lowcost/controller/UserController.java

```diff
@@ -67,15 +71,46 @@ public String updateUser(@RequestParam Map<String, String> params, Model model)
                         .build());
         if (user == null) {
             model.addAttribute("message", "No such user or it has been deleted!");
-        } else {
+        }
+        if(params.get("userUpdate").equals("fromUser")){
+            model.addAttribute("sessionUser", user);
+            return "redirect:/tickets/my-tickets";
```

Comment:
All parts of endpoints can be reused (for example my-tickets). All parts of all endpoints can be moved to separate class... But it will new task